### PR TITLE
Add timestamps to image overlay

### PR DESCRIPTION
**Pull request by lethosor**

### CODE REVIEW

File: web/src/components/layout.js [hunk 1]
```diff
@@ -72,6 +75,16 @@ class Layout extends React.Component {
       })
   }
 
+  openOverlay(overlayData) {
+    console.log('openOverlay', {overlayData})
```
Comment (benjaminpkane) [inline web/src/components/layout.js]: Remove?

File: web/src/styles/global.css [hunk 2]
```diff
@@ -66,6 +66,14 @@ img {
   width: 100%;
   height: 100%;
 }
+.image-overlay .image-timestamp {
+  position: absolute;
+  top: 5px;
+  left: 5px;
+  padding: 0 5px;
+  color: var(--gray-dark1);
+  background-color: var(--gray-light3);
```
Comment (benjaminpkane) [inline web/src/styles/global.css]: Could you make the background match the close button background (white)?

**Review by benjaminpkane — APPROVED** [review]
LGTM! Some minor comments.

File: web/src/styles/global.css (hunk 2 above)
Comment (lethosor) [reply web/src/styles/global.css]: The close button is actually `var(--gray-light2)`

Comment (lethosor) [reply web/src/styles/global.css]: Made them match

**Pull request merged**